clients/ptranslator/src/test/java/org/hawkular/metrics/clients/ptrans/data/ServerDataHelper.java

```diff
@@ -62,7 +61,7 @@ public ServerDataHelper(String tenant) {
         ObjectMapper objectMapper = new ObjectMapper();
 
         HttpURLConnection urlConnection = (HttpURLConnection) new URL(findGaugeMetricsUrl).openConnection();
-        urlConnection.setRequestProperty(String.valueOf(TENANT_HEADER_NAME), tenant);
+        urlConnection.setRequestProperty("Hawkular-Tenant", tenant);
```

Comment:
This constant was mostly used in production code. But since we'll now 
rely on vertx-hawkular-metrics for forwarding metrics, it is no longer 
needed.

I can add it back in test code, if that's a big concern.

Le 07/07/2016 à 19:19, Stefan Negrea a écrit :
> In
> clients/ptranslator/src/test/java/org/hawkular/metrics/clients/ptrans/data/ServerDataHelper.java
> <https://github.com/hawkular/hawkular-metrics/pull/529#discussion_r69948177>:
>
>> @@ -62,7 +61,7 @@ public ServerDataHelper(String tenant) {
>>          ObjectMapper objectMapper = new ObjectMapper();
>>
>>          HttpURLConnection urlConnection = (HttpURLConnection) new URL(findGaugeMetricsUrl).openConnection();
>> -        urlConnection.setRequestProperty(String.valueOf(TENANT_HEADER_NAME), tenant);
>> +        urlConnection.setRequestProperty("Hawkular-Tenant", tenant);
>
> While I do not expect the actual header name to change, why go this
> route here and in subsequent classes?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hawkular/hawkular-metrics/pull/529/files/f1db90b1cb3cdaebf6be5bfffd50a71e8182e2bc..0238f7d7e79b5236c6acebd6e6e7c199c51a6d42#r69948177>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABbltpYegboTLaR7xgnpTi2v66b9M0L_ks5qTTUqgaJpZM4JGMbv>.
>

-- 
Thomas Segismont
JBoss ON Engineering Team